init-odkx-sync-endpoint.py

```diff
@@ -48,65 +49,79 @@ def run_interactive_config():
 
     while True:
         print("Would you like to enforce HTTPS? We recommend yes.")
-        enforce_https = input("enforce https [(Y)/n]:")
+        enforce_https = input("enforce https [(Y)/n]:").lower().strip()
         if enforce_https == "":
             enforce_https = "y"
-            enforce_https = enforce_https.lower().strip()[0]
+            enforce_https = enforce_https[0]
```

Comment:
Why the index access here? I think it should be on line 52 instead? The intention I think was to allow someone to type "y" or "yes".

Reply:
@matt9j yeah, the logic is kinda broken when the string is non-empty but not y or n, i.e 'yes' or 'no'. It'll keep retrying since the loop won't exit.
However, I didn't change the logic so it's out of scope of this PR. Would you still like me to fix it here or in another PR?

Reply:
Leaving it here is okay.